### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.7 es 4.10 (30 ottobre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#306] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/d69aaebf-c025-4542-a687-3ab9c3137abe)

**Comment by marcoseverini** [comment]
credo
![immagine](https://github.com/user-attachments/assets/965defc0-d033-409c-ae86-abd50b0095d0)

**Reply by alemilos** [reply]
Penso che A debba essere Px(1) U Px(2) U Px(3) = 1/3

**Comment by rimaout** [comment]
<img width="1032" alt="image" src="https://github.com/user-attachments/assets/3be23956-90a9-4a82-8fdd-9c5b658461b7" />
